model.py

```diff
@@ -16,34 +16,48 @@ def mapper(value):
     return CLASS_MAP[value]
 
 def create_model(input_shape, num_classes):
-    model = keras.models.Sequential([
-        # NOTE the input shape is the desired size of the image with 3 bytes color
-        # This is the first convolution
-        # With 64 filters and a kernel_size of (3, 3)
-        # DOCS: https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv2D
-        keras.layers.Conv2D(64, (3,3), activation='relu', input_shape=input_shape),
-        keras.layers.MaxPooling2D(2, 2),
-        # The second convolution
-        keras.layers.Conv2D(64, (3,3), activation='relu'),
-        keras.layers.MaxPooling2D(2,2),
-        # The third convolution
-        keras.layers.Conv2D(128, (3,3), activation='relu'),
-        keras.layers.MaxPooling2D(2,2),
-        # The fourth convolution
-        keras.layers.Conv2D(128, (3,3), activation='relu'),
-        keras.layers.MaxPooling2D(2,2),
-        # The fifth convolution
-        keras.layers.Conv2D(128, (3,3), activation='relu'),
-        keras.layers.MaxPooling2D(2,2),
-        # Flatten the results to feed into a DNN
-        keras.layers.Flatten(),
+#     model = keras.models.Sequential([
+#         # NOTE the input shape is the desired size of the image with 3 bytes color
+#         # This is the first convolution
+#         # With 64 filters and a kernel_size of (3, 3)
+#         # DOCS: https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv2D
+#         keras.layers.Conv2D(64, (3,3), activation='relu', input_shape=input_shape),
+#         keras.layers.MaxPooling2D(2, 2),
+#         # The second convolution
+#         keras.layers.Conv2D(64, (3,3), activation='relu'),
+#         keras.layers.MaxPooling2D(2,2),
+#         # The third convolution
+#         keras.layers.Conv2D(128, (3,3), activation='relu'),
+#         keras.layers.MaxPooling2D(2,2),
+#         # The fourth convolution
+#         keras.layers.Conv2D(128, (3,3), activation='relu'),
+#         keras.layers.MaxPooling2D(2,2),
+#         # The fifth convolution
+#         keras.layers.Conv2D(128, (3,3), activation='relu'),
+#         keras.layers.MaxPooling2D(2,2),
+#         # Flatten the results to feed into a DNN
+#         keras.layers.Flatten(),
+#         keras.layers.Dropout(0.5),
+#         # 512 neuron hidden layer
+#         keras.layers.Dense(512, activation='relu'),
+#         keras.layers.Dense(num_classes, activation='softmax')
+#     ])
+    base_model = VGG16(weights='imagenet',include_top=False,input_shape=(255,255,3))
```

Comment:
what were the results of this?

Reply:
yeah the results were better than before. Not all images are classified as rock anymore! But it confuses between paper and scissor. 
![scissor_works](https://user-images.githubusercontent.com/48328081/89010477-45207c00-d32c-11ea-82fc-8fe2838b32cb.png) --> (1st image) correctly predicted image of scissor!

![paper_not_work](https://user-images.githubusercontent.com/48328081/89010729-c6780e80-d32c-11ea-8599-044357d25195.png) --> (2nd image) Incorrectly predicted image of an actual paper hand image as being scissor. many other image got same result, very few with correct paper prediction.

Reply:
should probably train it with more images then I don't see any other possible solution.
the min no of training set would be 10k and validation set about 2k.
and moreover, this code doesn't split the dataset into training and validation, so it has nothing to validate against or give us proper results.

Reply:
But overall, using the VGG16 model, got a definite increase in prediction accuracy, but there is still some uncertainty in the model. I wanted to implement data augmentation, but I think I would have to change a lot of code in the model file., to implement it. 
AsI have done data augmentation and then using flow_from_directory() using the augmented functions in previous projects of mine, where I used the folder names for classification; but I don't know how to implement it here directly, with 800 images lumped together.

Reply:
yeah, your right. I'll try to create a validation set, i think as you said it might help gauge the model.

> should probably train it with more images then I don't see any other possible solution.
> the min no of training set would be 10k and validation set about 2k.
> and moreover, this code doesn't split the dataset into training and validation, so it has nothing to validate against or give us proper results.